README.md

```diff
@@ -2,71 +2,113 @@
+# Clone project
+git clone git@github.com:knocklabs/knock-node-example-app.git
+cd knock-node-example-app
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+# Install deps
+yarn
+```
+
+**2. Create your `.env` file and copy the content from `.env.sample`**
 
-## Environment variables
+Set the necessary environment variables. To get started with triggering workflows and the in-app feed, you must have variables set for the following:
 
-See the `.env.sample` and `.env.test.sample` files for the env variables needed.
+- `DATABASE_URL`
+- `KNOCK_API_KEY`
+- `BLITZ_PUBLIC_KNOCK_CLIENT_ID`
+- `BLITZ_PUBLIC_KNOCK_FEED_ID`
 
 Here's more context on where to find the Knock variables:
 
 - [Knock API keys](https://docs.knock.app/developer-tools/api-keys)
 - [Knock in-app feed channel ID](https://docs.knock.app/in-app-ui/react/feed#getting-started)
 - [Knock Slack channel ID](https://docs.knock.app/integrations/chat/slack/building-oauth-flow#how-to-set-slack-channel-data-in-knock)
 
-## Using the example app
+**3. Set up your database:**
 
-Before you run the example app locally, you'll need to [sign up for a Knock account](https://dashboard.knock.app/signup) to configure channels and workflows for the app to work as expected.
+```
+# Migrate
+yarn blitz prisma migrate dev
 
-Once you've created your Knock account you will need to create:
+# Seed
+yarn blitz db seed
+```
 
-1. Email and in-app feed channels
-2. A workflow in Knock with the key `welcome`. It should include an email step for your welcome email template.
-3. A workflow in Knock with the key `new-comment`. It should include: an in-app feed step and an email step. On these step's templates you can use the following variables which the example app is already configured to send as arguments on the workflow trigger call:
-   1. `comment_content`: The content of the created comment.
-   2. `asset_name`: The name of the commented asset.
-   3. `asset_url`: The url of the commented asset.
-   4. `project_name`: The name of the project that the comment asset belongs to.
-4. A workflow in Knock with the key `new-asset`. It should include: an in-app feed step and an email step. On these step's templates you can use the following variables which the example app is already configured to send as arguments on the workflow trigger call:
-   1. asset_url: The url of the commented asset.
-   2. project_name: The name of the project that the comment asset belongs to.
+## Running locally
+
+After you complete the setup steps, you can start running the app.
+
+```
+yarn blitz dev
+```
 
-Once you've seeded your Knock account with `yarn blitz db seed` and created the above workflows in Knock, you can use the app locally and trigger Knock workflows to send notifications from the example app.
+This will start your application at http://localhost:3000. You can take a look at the seed users credentials in the seed file in order to log in; all starter passwords are `password`.
```

Comment:
I think we should include an email here to make it quicker to get started, while still referencing the seed file.